src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -263,6 +266,18 @@ private class ParameterImpl(private val bean: ParameterBean) : Parameter {
         get() = bean.own
 }
 
+private class TriggerImpl(private val bean: TriggerBean) : Trigger {
+
+    override fun fetchDependsOnBuildConfiguration():
```

Comment:
Why do you use functions with names 'fetch...' for these properties? They don't perform REST requests and simply return already fetched data, so it would be better to use simple properties for them (like we do for other properties in this library).